include/nbl/core/alloc/LinearAddressAllocator.h

```diff
@@ -20,25 +20,15 @@ class LinearAddressAllocator : public AddressAllocatorBase<LinearAddressAllocato
         typedef AddressAllocatorBase<LinearAddressAllocator<_size_type>,_size_type> Base;
     public:
         _NBL_DECLARE_ADDRESS_ALLOCATOR_TYPEDEFS(_size_type);
-
-        #define DUMMY_DEFAULT_CONSTRUCTOR LinearAddressAllocator() : bufferSize(invalid_address), cursor(invalid_address) {}
-        GCC_CONSTRUCTOR_INHERITANCE_BUG_WORKAROUND(DUMMY_DEFAULT_CONSTRUCTOR)
-        #undef DUMMY_DEFAULT_CONSTRUCTOR
-
+       
         virtual ~LinearAddressAllocator() {}
-
+        
+        LinearAddressAllocator() : bufferSize(invalid_address), cursor(invalid_address) {}
```

Comment:
I did not mean or intend for the address allocators to have default constructors, see
https://github.com/Devsh-Graphics-Programming/Nabla/blob/6a17dc80e50be0fee6114e2efd2408fbb223968a/include/nbl/macros.h#L114

Reply:
I thought I undid that

Reply:
sorry git diff view mesed up for me, you're right